src/LogViewer/chunk-worker.js

```diff
@@ -115,7 +115,7 @@ const update = (response) => {
     lineCounts = lineCounts.slice(overage);
   }
 
-  self.postMessage(JSON.stringify({ type: 'update', chunkHeights }));
+  self.postMessage(JSON.stringify({ type: 'update', chunkHeights, offset, LINE_CHUNK }));
```

Comment:
`LINE_CHUNK` is a constant to this file, but being something passed to `LogViewer` makes it not constant to the other file, so it should not be uppercased here. Something like:

```js
{  lineChunk: LINE_CHUNK  }
```